pkg/sql/catalog/bootstrap/initial_values.go

```diff
@@ -393,4 +356,298 @@ const (
 ]`
 	// nonSystem222SHA256 is the SHA-256 hash of the above constant.
 	nonSystem222SHA256 = `49b3e2d0bb0d18d711d2fff94b0c11db2e5caecf3c4fcc27d9bd277694647afd`
+
+	// system222 is the hard-coded string representation of the
+	// initial values for a system tenant as produced by the 23.1
+	// release binary.
+	//
+	// This string literal was generated by copying over the expected results
+	// for the TestInitialValuesToString test in the release-23.1 branch.
+	system231 = `[{"key":"04646573632d696467656e","value":"01c801"}
```

Comment:
We could consider using `embed` to make the actual source code file lighter, especially with LTS and us having to support multiple releases.